z_infra_provisioning/cloud_infra_center/ocp_upi/README.md

```diff
@@ -279,11 +279,13 @@ Others are **optional**, you can enable them and update value if you need more s
 | `os_bootstrap_ip` | \<bootstrap ip addr\> |'x.x.x.x, <br>**required** when `auto_allocated_ip` is false
 | `os_master_ip` | \<master ip list\>|'[x.x.x.x, x.x.x.x, x.x.x.x], <br>**required** when `auto_allocated_ip` is false
 | `os_infra_ip` |\<infra ip list\>|'[x.x.x.x, x.x.x.x, x.x.x.x], <br>**required** when `auto_allocated_ip` is false
+| `use_storage_provider` |false|(Boolean) true or false, if true then Openshft Container Platform will create the bootable volume
+| `volume_type_template` |\<volume template id\>|`openstack volume type list -c ID -f value`, <br>**required** when `use_storage_provider` is true
```

Comment:
suggest change volume_type_template to volume_type_id

community call it "volume type" while in ICIC we call it "storage template".

Reply:
thanks for your comments, it's already merged.